README.md

```diff
@@ -29,6 +29,12 @@ When taking a dependency on a Fluid Framework library, we recommend using a `^`
 While Fluid Framework libraries may use different ranges with interdependencies between other Fluid Framework libraries,
 library consumers should always prefer `^`.
 
+Note that when depending on a library version of the form `2.0.0-internal.x.y.z`, called the Fluid internal version scheme,
```

Comment:
We don't want these changes about the internal package versions. I think these were re-added by mistake. You'll want to re-run `pnpm i` from the root and re-run `build:readme`, and these changes should go away.

Reply:
Let me revert the last commit and so them again

Reply:
You may need to `git clean -dxf` and rerun 😕